rocAL/rocAL/CMakeLists.txt

```diff
@@ -38,6 +38,10 @@ set(BOOST_COMPONENTS filesystem system)
 find_package(Boost COMPONENTS ${BOOST_COMPONENTS} QUIET)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads QUIET)
+find_library(libsnd_LIBS
```

Comment:
Check if find_package or find_library is the correct use here

Reply:
find_package is used to locate and configure CMake packages that have CMake-specific configuration files.

Here find_library seems to be the right usage